build.gradle

```diff
@@ -1,27 +1,37 @@
 plugins {
     id 'java'
     id 'maven-publish'
+    id "net.kyori.blossom" version "1.1.0"
 }
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```

Comment:
This needs to also set -release 8 (see another project) as its building on never java.

Reply:
Ah, so like
```gradle
if (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_1_8) {
    tasks.withType(JavaCompile) {
        options.release = 8
    }
}
```
?

Reply:
```gradle
tasks.withType(JavaCompile).configureEach {
	if (JavaVersion.current().isJava9Compatible()) {
		 it.options.release = 8
	}
}
```

Is slighly nicer, but yes.